packages/terminal-extension/schema/plugin.json

```diff
@@ -46,7 +46,7 @@
       "title": "Theme",
       "description": "The theme for the terminal.",
       "$ref": "#/definitions/theme",
-      "default": "dark"
+      "default": "inherit"
```

Comment:
I guess I'm just naturally a bit conservative about defaults: any reason to change this from "dark"?

Reply:
@ellisonbg What do you think?

Reply:
Good point. What about having another setting of "opposite" (dark terminal on the light theme) and having that as the default? The reason we have had a dark terminal on a light theme previously is to help users understand that it is different from other UIs that look similar (like a test editor).

Reply:
True. I'm not a fan on light terminal on dark theme. Give it a try. I'm ok either way 👌